src/utils/switch-site.ts

```diff
@@ -5,9 +5,9 @@ import { logout } from "./logout"
 import { queryClient } from "./query-client"
 
 const SITE_TO_USER_MAP: Record<SiteKey, string> = {
-  stitch: "rene@example.com",
-  luminara: "cecilia@example.com",
-  pug: "emily@example.com",
+  stitch: "cecilia@example.com",
+  luminara: "emily@example.com",
+  pug: "rene@example.com",
 }
```

Comment:
Fixes a bug where the product title and product category is completely mismatched in Shoppy. You'll notice in the current demos that the product titles doesn't really match the shop's title, and this is why.

PS. You will get the wrong image if you try to run the demo now. This is expected, because I put the wrong product images in the wrong shop all along 🤦🏻. I don't want to break the current Shoppy demo in production, so I will only run the script to update the product images immediately after this PR is released.

![CleanShot 2567-09-04 at 02 24 30@2x](https://github.com/user-attachments/assets/1fdc9b42-94dc-4ae4-90d2-b2a66388d94a)